pymc3/model.py

```diff
@@ -500,7 +505,8 @@ def Var(self, name, dist, data=None):
                     self.named_vars[v.name] = v
         else:
             var = ObservedRV(name=name, data=data,
-                             distribution=dist, model=self)
+                             distribution=dist, model=self,
+                             total_size=total_size)
```

Comment:
there's an implicit convenction that `model` is always the last kwarg. Not sure it matters but might as well stick to that.